pretty-hydra.el

```diff
@@ -218,10 +232,12 @@ See `pretty-hydra-define' and `pretty-hydra-define+'."
          (title     (plist-get body :title))
          (formatter (or (plist-get body :formatter)
                         #'identity))
+         (title-body-format-spec (plist-get body :title-body-format-spec))
          (quit-key  (plist-get body :quit-key))
-         (docstring (->> heads-plist
-                         (pretty-hydra--gen-body-docstring separator)
-                         (pretty-hydra--maybe-add-title title)
+         (docstring (->> (pretty-hydra--maybe-add-title
```

Comment:
Here I had to replace a bit of the `->>` threading by old-school sexp-nesting, because I added an optional argument to `pretty-hydra--maybe-add-title`. Therefore `docstring` can't be the last argument to it anymore. I have no experience with `dash.el` threading macros so if you know a prettier way how to solve this, let me know.

Reply:
Implemented your suggestion in 0f9a68582478e86f9a643f9626ce5f16ecce3468